arista-dcs-7124S/workflows/deploy-startup-config-graph.json

```diff
@@ -0,0 +1,29 @@
+{
+    "friendlyName": "Deploy Arista switch",
+    "injectableName": "Graph.Switch.Arista.Deploy",
+    "options": {
+        "deploy-config": {
```

Comment:
Right. Also changing it to "deploy-startup-config" in the config.json file. Thanks!